lib/commands/roku.js

```diff
@@ -149,10 +149,10 @@ export async function roku_appUI (stripOuterTags = false) {
 
 /**
  * @this RokuDriver
- * @param {{appId: string}} opts
+ * @param {{appId: string, contentId?: string, mediaType?: string}} opts
```

Comment:
Maybe it'd be a good idea to create an `ActivateAppOptions` type and refer to it in both places?